apis/v1alpha1/gateway_types.go

```diff
@@ -430,6 +430,8 @@ type RouteBindingSelector struct {
 	// default.
 	//
 	// Support: Core
+	//
+	// +optional
```

Comment:
I think an optional field should be `omitempty`

Reply:
But it has a default, so it will technically never be empty.